packages/v4/patternfly-docs/pages/home.js

```diff
@@ -131,7 +131,7 @@ const HomePage = () => (
                   <Text
                     className="pf-v5-u-pl-0 pf4-c-link__text ws-mdx-a"
                   >
-                    <Link to="/demos/primary-detail" aria-label="view demos">
+                    <Link to="/patterns/primary-detail" aria-label="view demos">
```

Comment:
@mcarrano Is this the best page to link to? Also, they are now called "Patterns" in the navigation (but are tagged as "demo") so should we change this to "View patterns"?

Reply:
@srambach I think this is OK to just change the link location. I see this is coming from the home page where we have some outdated content. But we are replacing the home page anyway. 

As for the demo tag, it is intended to apply to things that include demos. All patterns may not include demos.